yaks/spec/acceptance/json_shared_examples.rb

```diff
@@ -1,19 +1,52 @@
-RSpec.shared_examples_for 'JSON output format' do |yaks, format, name|
-  let(:input)  { load_yaml_fixture(name) }
-  let(:output) { load_json_fixture("#{name}.#{format}") }
+RSpec.shared_examples_for 'JSON Writer' do |fixture_name|
+  describe 'Yaks::Resource => JSON' do
+    let(:object)  { load_yaml_fixture(fixture_name) }
+    let(:json_fixture) { load_json_fixture("#{fixture_name}.#{format_name}") }
+    let(:serialized) {
+      yaks_config.call(object, hooks: [[:skip, :serialize]], format: format_name)
+    }
 
-  subject { yaks.call(input) }
+    # before do
```

Comment:
I left it in deliberately, found it can be quite helpful when debugging these tests.